roles/hostedbm/README.md

```diff
@@ -0,0 +1,91 @@
+hostedbm
+=========
+This role creates hosted guest cluster with BM workers.
+
+
+Requirements
+------------
+
+* Access to an OpenShift hosting cluster
+* Properly configured kubeconfig file for the hosting cluster access
+* Existing CatalogSource.
+* The OpenShift hosting cluster should have a way to configure Persistent Volumes (PVs), which could be through the ODF (OpenShift Data Foundation), LVM Storage Operator, or any other suitable method.
+* ACM with MCE installed on the hosting cluster
+* Metallb operator installed on the hosting cluster
+* Metallb configured on the hosting cluster - MetalLB, L2Advertisement, and IPAddressPool configured for range of IP to be used as `APIServer` for hosted clusters.
+* AgentServiceConfig and Provisioning configured on the hosting cluster
```

Comment:
Can the role validate that ACM, MCE, ASG, and provisioning are in place?

Reply:
Thank you for your suggestion.

Actually, we internally reuse this code in our automation. Before calling this hostedbm role, we invoke another role that sets up ACM, configures LVMS storage, and installs and configures MetalLB. We validate the presence of ACM, ASG, and provisioning objects within our ACM installation role.

In my humble opinion, validating everything mentioned in the requirements for this role would make it lengthy, time-consuming, and would also result in duplicated validation across different roles.

I assume you have roles for ACM installation where you perform these object validations.

But if that's the standard in this repo, I don't have strong objections to adding those validations here. Please let me know what you think.

Reply:
That works, yes. The other roles have the required validations.